### PR TITLE
final changes

### PR DESCRIPTION
**Pull request by Favy-source**
I've fixed all messy work @konfirmed

### CODE REVIEW

**Comment by konfirmed** [comment]
@Favy-source 
image

**Comment by konfirmed** [comment]
@Favy-source 
image

**Comment by konfirmed** [comment]
@Favy-source 
image

**Review by Favy-source — COMMENTED** [review]
Ready for merging @konfirmed

**Comment by Favy-source** [comment]
Its good to go now @konfirmed

**Review by konfirmed — APPROVED** [review]

File: pages/culture.mdx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 
 Overall, the emergence of Igbo popular culture and the impact of globalization have had a significant impact on Igbo society and culture. While these changes have brought many new opportunities and challenges, they have also helped to shape the Igbo society and culture that we know today.
 
 Igbo people has a dynamic and fascinating cultural heritage that says lots about them, and most of the Igbo people are Christians. Their cultures are further divided into many groups, due to dialects and boundaries among the eastern states in particular and a good sample of this is their traditional way of welcoming visitors, which is usually offering kola to guests, even before they made their mission known.
-#### The role of traditional Igbo religion
+
+## Traditional Igbo Religion
+The role of traditional Igbo religion
```
[the remark this answers is not in the document]
Comment (konfirmed) [reply pages/culture.mdx:13]: @Favy-source you did not make the change I asked for here

[the remark this answers is not in the document]
Comment (konfirmed) [reply pages/culture.mdx:13]: @Favy-source you did not make the change here

**Review by konfirmed — COMMENTED** [review]
Please make the required changed

**Pull request merged**